packages/btc/src/api/sendRgbppUtxos.ts

```diff
@@ -0,0 +1,169 @@
+import { helpers, Hash, RawTransaction, RPC } from '@ckb-lumos/lumos';
+import { InitOutput, TxAddressOutput } from '../transaction/build';
+import { ErrorCodes, TxBuildError } from '../error';
+import { DataSource } from '../query/source';
+import { Utxo } from '../types';
+import { bitcoin } from '../bitcoin';
+import { RGBPP_UTXO_DUST_LIMIT } from '../constants';
+import { calculateCommitment } from '../ckb/commitment';
+import { unpackRgbppLockArgs } from '../ckb/molecule';
+import { getCellByOutPoint } from '../ckb/rpc';
+import { sendUtxos } from './sendUtxos';
+
+export async function sendRgbppUtxos(props: {
+  ckbVirtualTx: RawTransaction;
+  paymaster: TxAddressOutput;
+  commitment: Hash;
+  tos?: string[];
+
+  ckbNodeUrl: string;
+  rgbppLockCodeHash: Hash;
+  rgbppTimeLockCodeHash: Hash;
+  rgbppMinUtxoSatoshi?: number;
```

Comment:
```
  ckbNodeUrl: string;
  rgbppLockCodeHash: Hash;
  rgbppTimeLockCodeHash: Hash;
```
I suggest that we should remove the parameters and the commitment calculation has been done in CKB package

Reply:
1. Yes, I believe the commitment validation logic can be refactored using the `@rgbpp-sdk/ckb` lib 👍. 
2. However, the props you mentioned may be impossible to remove due to the needs of transaction construction. These props are used to query `Cell` from `OutPoint` in the inputs, and I have to check which input is an `RgbppLock` cell and which output is an `RgbppTimeLock` cell.

Reply:
OK, I see. 

But you can get the RGBPP lock script and BTC time lock script information from the CKB package(src/constants). You can use the Collector of CKB package to query cells.

I think the parameters are not necessary for developers and they make the function usage more complicated.